packages/mobile/src/backup/utils.ts

```diff
@@ -42,3 +47,28 @@ export function getWordlist(language: string | null) {
   }
   return wordlist
 }
+
+// TODO(Rossy) Remove after the next alfa testnet reset
+export async function getMnemonicFromStorage(): Promise<string | null> {
```

Comment:
nit: this could be named `getStoredMnemonic` as I feel `getMnemonicFromStorage` seems to indicate it only reads from `AsyncStorage`.
What do you think?

Reply:
Agreed, will rename now